README.md

```diff
@@ -30,11 +30,11 @@ Don't name arguments or flags with names that contain underscore symbols, becaus
 
 <!-- usage -->
 ```sh-session
-$ npm install -g @fluencelabs/cli
+$ npm install -location=global @fluencelabs/cli
 $ fluence COMMAND
 running command...
-$ fluence (--version)
-@fluencelabs/cli/0.0.2 linux-x64 node-v16.14.0
+$ fluence --version
```

Comment:
I think creators of the framework decided this auto-generated makes sense because you can also just say `fluence` (without using --version flag) and it will show you the help text, so in my personal opinion it's ok to leave it as it is